block/stream/output/state_changes.proto

```diff
@@ -795,6 +805,16 @@ message MapChangeValue {
          * A roster value.
          */
         com.hedera.hapi.node.state.roster.Roster roster_value = 16;
+
+        /**
+         * A list of scheduled ids.
+         */
+        proto.ScheduleIdList schedule_id_list_value = 17;
```

Comment:
I _believe_ this can **replace** field number 8, as we do not store lists of full schedules any longer.

Reply:
Still used as above

Reply:
Similar, this is a block stream item, and shouldn't be part of any upgrade migration...